docs/nodes/configuration.md

```diff
@@ -34,25 +34,25 @@ like the file below, however, double check by inspecting the
 proxy-app = "tcp://127.0.0.1:26658"
 
 # A custom human readable name for this node
-moniker = "anonymous"
+moniker = "Fergalicious.local"
 
-# If this node is many blocks behind the tip of the chain, BlockSync
-# allows them to catchup quickly by downloading blocks in parallel
-# and verifying their commits
-fast-sync = true
-
-# Mode of Node: full | validator | seed (default: "validator")
-# * validator node (default)
+# Mode of Node: full | validator | seed
+# * validator node
 #   - all reactors
 #   - with priv_validator_key.json, priv_validator_state.json
-# * full node 
+# * full node
 #   - all reactors
 #   - No priv_validator_key.json, priv_validator_state.json
 # * seed node
 #   - only P2P, PEX Reactor
 #   - No priv_validator_key.json, priv_validator_state.json
 mode = "validator"
 
+# If this node is many blocks behind the tip of the chain, FastSync
```

Comment:
how many is enough?

Reply:
not sure why this is an option in the config TBH. The node should always fall back to fast sync if behind. @cmwaters do you know why this is a thing?

Reply:
If you're running blocksync then the termination conditions are:
1) you verify the block of the highest height amongst all your peers
2) a global timeout

If you have this switched to true you will always attempt block sync first before switching to consensus. At the moment, if you fall behind in consensus you will not go back into blocksync but remain in consensus. This is an issue yet to be resolved and requires some sort of reactor orchestration.

Reply:
is there any case a user would set this to false?

Reply:
@cmwaters Is the global timeout a param somewhere?

Reply:
> is there any case a user would set this to false?
If they just wanted to catch up via consensus I guess

Reply:
> @cmwaters Is the global timeout a param somewhere?

Just checked now and it's not so much a global timeout rather termination is when the last response from a peer is greater than 15 seconds (and yes the 15 seconds part is hardcoded)